test/errors/read.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { newUnibeautify, Beautifier } from "unibeautify";
+import beautifier from "../../src";
+import { raw } from "../utils";
+// tslint:disable:mocha-no-side-effect-code
+const fs = require("fs");
+const originalReadFile = fs.readFile;
+beforeEach(() => {
+  fs.readFile = jest.fn((filepath, cb) => cb(new Error("Read file failed")));
```

Comment:
Any way to use `Jest.mock` here?

Reply:
I tried this, but always received `Cannot read property 'prototype' of undefined` error. Guessing it overrides the entire fs library instead of just `readFile`.  I'll try looking more into it.
```
jest.mock("fs", () => ({
  readFile(filepath: any, callback: any) {
    return callback(new Error("Create file failed"));
  },
}));
```

Reply:
Hmm that looks correct to me. Next time share the full error and stack trace. Let's try to figure out the proper way with Jest once and for all. 👍 😃

Reply:
`TypeError: Cannot read property 'prototype' of undefined`

Reply:
... Any line information? :P Any more context at all?

Reply:
Can you get on chat?

Reply:
👍